fiftyone/core/odm/__init__.py

```
@@ -63,6 +63,7 @@
 from .embedded_document import (
     BaseEmbeddedDocument,
     DynamicEmbeddedDocument,
+    DynamicEmbeddedDocumentException,
```

Ruff (0.8.2)

`.embedded_document.DynamicEmbeddedDocumentException` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)